autofit/non_linear/mcmc/abstract_mcmc.py

```diff
@@ -3,35 +3,59 @@
 from autoconf import conf
 from autofit.non_linear import samples as samp
 from autofit.non_linear.abstract_search import NonLinearSearch
+from autofit.non_linear.mcmc.auto_correlations import AutoCorrelationsSettings, AutoCorrelations
 
 
 class AbstractMCMC(NonLinearSearch):
+
+    def __init__(
+            self,
+            name="",
+            path_prefix="",
+            prior_passer=None,
+            initializer=None,
+            auto_correlations_settings = AutoCorrelationsSettings(),

```

Comment:
Be careful using mutable objects as default arguments. It would be better to use `or` in the constructor implementation